wrapper/pom.xml

```diff
@@ -124,29 +125,41 @@
             <artifactId>opentelemetry-aws-lambda-1.0</artifactId>
             <version>${otel.instrumentation.version}</version>
         </dependency>
+        <!-- OTEL SDK -->
         <dependency>
             <groupId>io.opentelemetry</groupId>
             <artifactId>opentelemetry-sdk</artifactId>
             <version>${otel.version}</version>
         </dependency>
         <dependency>
             <groupId>io.opentelemetry</groupId>
-            <artifactId>opentelemetry-extension-trace-propagators</artifactId>
-            <version>${otel.version}</version>
+            <artifactId>opentelemetry-sdk-extension-autoconfigure</artifactId>
+            <version>${otel.version.alpha}</version>
```

Comment:
we should really switch to using the otel BOMs to make sure all our versions stay in sync.